### PR TITLE
Bump substrate interface to 0.13.10

### PR DESCRIPTION
**Pull request by yabirgb**
Issue with substrate should be fixed https://github.com/polkascan/py-substrate-interface/issues/124#issuecomment-882411729

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ gevent-websocket==0.10.1
 wsaccel==0.6.3  # recommended for acceleration of gevent-websocket. But abandoned.
 web3==5.21.0
 pysqlcipher3==1.0.4
-requests==2.25.1
+requests==2.26.0
```
Comment (LefterisJP) [inline requirements.txt:7]: I already commented in their repo that this is not good. We can't be limited by their requirements and have to upgrade when they do.

https://github.com/polkascan/py-substrate-interface/commit/c9e48460c4cd5b2d00e5e717eecba4b581e3c166#r53677476

**Review by LefterisJP — COMMENTED** [review]
Hey @yabirgb so please do check out what could be wrong with this. not sure why it fails still. Try it locally.

**Comment by yabirgb** [comment]
The issue seems to be that recently they added packaging as dependency https://github.com/polkascan/py-substrate-interface/commit/c9e48460c4cd5b2d00e5e717eecba4b581e3c166#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R185 and while packaging it can't import it. Trying more things as what I tried didn't seem to work

**Comment by yabirgb** [comment]
Remoce `packaging` from here https://github.com/rotki/rotki/blob/develop/rotkehlchen.spec#L96 makes the build run. As we commented in discord this shouldn't be packaged

**Comment by yabirgb** [comment]
Dependency was updated again and now it finally allows us to build and we upgraded requests. 

- Green builds https://github.com/rotki/rotki/actions/runs/1049244022

File: requirements.txt (hunk 1 above)
Comment (kelsos) [reply requirements.txt:7]: Do we still want to upgrade requests here?

**Review by kelsos — APPROVED** [review]
Ok @yabirgb I am approving the only question is do we really want to update requests or is it a leftover?

**Comment by yabirgb** [comment]
@LefterisJP Lefteris mentioned that indeed we wanted to do it https://github.com/polkascan/py-substrate-interface/commit/c9e48460c4cd5b2d00e5e717eecba4b581e3c166#r53677476

File: requirements.txt (hunk 1 above)
Comment (LefterisJP) [reply requirements.txt:7]: yes it's okay. But need to test in case it breaks anything.

**Comment by kelsos** [comment]
The e2e test seems to be the one that was partially fixed in bugfixes (with the asset db updates) 
The backend test I have no idea. Is it safe to merge?

**Pull request merged**